integrations/langchain/src/databricks_langchain/vectorstores.py

```diff
@@ -242,7 +242,9 @@ def __init__(
             ) from e
 
         try:
-            self.index = VectorSearchClient(**(client_args or {})).get_index(
+            client_args = client_args or {}
+            client_args["disable_notice"] = True
```

Comment:
Can we do something like this:

```
client_args = {"disable_notice": True, **(client_args or {})}
```

So here we are setting it as default but we let users override it if needed rather than overriding what users pass in?